modyn/supervisor/internal/grpc_handler.py

```diff
@@ -544,18 +495,79 @@ def start_evaluation(self, trained_model_id: int, pipeline_config: dict) -> dict
 
         return evaluations
 
+    @staticmethod
+    def _get_evaluate_model_request(
```

Comment:
maybe rename to prepare request?

Reply:
Sure, done. I tested the pipeline end-to-end and everything seemed to work out.